DSharpPlus/Extensions/ServiceCollectionExtensions.InternalSetup.cs

```diff
@@ -140,4 +143,18 @@ DiscordIntents intents
 
         return serviceCollection;
     }
+
+    private static IServiceCollection RegisterBestDecompressor(this IServiceCollection services)
```

Comment:
Perhaps rename this to `RegisterDefaultDecompressor()` as `Best` is subjective?

Reply:
currently writing the article on what we mean by "best"